app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class='move_down'>
+  <h2>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), class: '') do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+    <div class="input_label">
+      <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: :name %>
+    </div>
+
+    <div class="input_label">
+      <%= f.email_field :email, autocomplete: "email", placeholder: :email %>
+    </div>
+
+    <div class="input_label">
+      <%= f.password_field :password, autocomplete: "new-password", placeholder: "#{:password}" %></br>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+    </div>
+
+    <div class="input_label">
+      <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: :password_confirmation %>
+    </div>
+
+    <div class="actions">
+      <%= f.submit "Sign up", class: 'add_btn' %>
+    </div>
+  <% end %>
+
+  <%= render "devise/shared/links" %>
+<div>
```

Comment:
- [ ] as one of the requirements is to follow the design provided you should follow colors, font-family, font-size and composition, and space between elements.
- Screenshot of the Requirements 
<img width="451" alt="Screenshot 2023-03-23 at 11 59 57" src="https://user-images.githubusercontent.com/61790587/227199009-9390b89c-7adb-4af5-9518-9a98992826e7.png">
- Font provided with colors 
<img width="437" alt="Screenshot 2023-03-23 at 12 00 24" src="https://user-images.githubusercontent.com/61790587/227200012-15900ad2-19ce-48b3-8281-329881d352f9.png">
<img width="498" alt="Screenshot 2023-03-23 at 12 00 32" src="https://user-images.githubusercontent.com/61790587/227200028-c1ab6779-e746-4065-a35d-c328d6734df8.png">


- [ ] The signup page should be the same as the one in the design by styling the background, font , colors , the inputs 
- yours 
<img width="333" alt="Screenshot 2023-03-23 at 12 01 31" src="https://user-images.githubusercontent.com/61790587/227200175-9c734426-bf51-444f-b8d7-f547476b23be.png">
- Design 
<img width="230" alt="Screenshot 2023-03-23 at 12 01 50" src="https://user-images.githubusercontent.com/61790587/227200221-c18195c5-d95d-44f7-8fcc-f0d9d7328fe1.png">